docs/rules-problems-solutions.txt

```diff
@@ -0,0 +1,53 @@
+
+
+Following added solutions to improve the matchmaking system. Shit ideas are enclosed by [.....]
+
+
+
+
+1) Sucessful vote to start a league match requires 100% majority. This is to make sure the game can start without delay,
+  and other trouble like being afk while everyone warms up.
+
+    
+2) Leaving the game and not rejoining within 5 minutes earns you the same ranking as if you've lost the game and performed very poorly.
+
+
+
+3) A new player that comes and replaces someone gets the same points ingame as the worst player on the team.
+
+For example: If Sonix replaces Player A, the topscorer of team blue, Sonix will pick the game up with the same ingame points as Player D,
+ who is bottom scorer. He will not pick up where the leaver and topscorer A left. Rinse and repeat if this occasion happens multiple times.
+
+ This is quite experimental, needs beta testing.
+
+
+
+4) For capgames/Clans (not for TDM): If a league game is in progress, and someone leaves, first of all the game is fully frozen/ "paused". 
+   If a substitute is not found within 5 minutes countdown again, the game will be erased from database. ]
+
+---> Righfully according to Shiki, this is not possible with the current implementation.
+
+[Now, for capgames, in some special circumstances, people can fuck around on purpose and make this a pain in the ass, if they decide just to 
+leave and not come back. That's why we should keep the demo (if you can get it to work shiki) of these games, inspect the case by moderators, 
+and then take appropriate measures against the user. For clanwars, i see this as a minor problem at best. Leaving games should be simply punished, 
+like in every other online game.]
+
+---> We can never get this to work because of the required space to save all of the server side demos. NEEDS BETTER SOLUTION:
```

Comment:
Looks like redundant comment.
Can you style the solutions like pros-cons instead?